.circleci/config.yml

```diff
@@ -44,29 +44,55 @@ jobs:
 
 
 
-  e2eTestCurrentVersionPR:
+  e2eTestCurPRReboot:
     environment:
-      MINIKUBE_VERSION: v0.25.0
       TESTED_VERSION: "current"
+      TEST_DIR: "integration/test/reboot"
     <<: *e2eTest
 
-  e2eTestCurrentVersionMaster:
+  e2eTestCurMasterReboot:
     environment:
-      MINIKUBE_VERSION: v0.25.0
       TESTED_VERSION: "current"
+      TEST_DIR: "integration/test/reboot"
     <<: *e2eTest
 
-  e2eTestWIPVersionPR:
+  e2eTestWIPPRReboot:
     environment:
-      MINIKUBE_VERSION: v0.25.0
       TESTED_VERSION: "wip"
+      TEST_DIR: "integration/test/reboot"
     <<: *e2eTest
 
 
-  e2eTestWIPVersionMaster:
+  e2eTestWIPMasterReboot:
     environment:
-      MINIKUBE_VERSION: v0.25.0
       TESTED_VERSION: "wip"
+      TEST_DIR: "integration/test/reboot"
+    <<: *e2eTest
```

Comment:
Time to learn :) What does `<<: *e2eTest` mean?

Reply:
When you check the full file you see that there is a block defined which is reused. This is the syntax magic to say "reuse the block e2eTest". AFAIK this is some YAML magic.

Reply:
Yes, with node anchors you define the blocks you want to repeat http://www.yaml.org/spec/1.2/spec.html#id2785586 and with alias nodes you insert them http://www.yaml.org/spec/1.2/spec.html#id2786196. The `<<` is useful when you want to insert a map and not a specific node http://yaml.org/type/merge.html.